### PR TITLE
5_Types

### PR DESCRIPTION
**Pull request by laurayuwen**
5_types, unification.k, MetaK.java

### CODE REVIEW

**Comment by cos** [comment]
@laurayuwen, is this ready for merging?

**Comment by laurayuwen** [comment]
@cos Yes.

File: k-distribution/tutorial/1_k/5_types/lesson_9/lambda.k [hunk 1]
```diff
@@ -46,18 +58,18 @@ module LAMBDA
...
   rule <k> mu X:Id . E:Exp => (?T:Type -> ?T) E ~> tenv(TEnv) ...</k>
        <tenv> TEnv => TEnv[X <- ?T] </tenv>
 
   syntax KItem ::= Type "=" Type
-  rule T = T => .
+  rule T = T => .K
 
   syntax KItem ::= tenv(Map)
-  rule <k> T:Type ~> (tenv(TEnv) => .) ...</k>  <tenv> _ => TEnv </tenv>
+  rule <k> `T:Type ~> tenv(TEnv) => T` ...</k> <tenv> `_ => TEnv` </tenv>
```
Comment (cos) [inline k-distribution/tutorial/1_k/5_types/lesson_9/lambda.k]: should work without the second set of backticks

Comment (laurayuwen) [inline k-distribution/tutorial/1_k/5_types/lesson_9/lambda.k]: Yes, I think so. So as long as it works, the less backticks the better?

Comment (dwightguth) [inline k-distribution/tutorial/1_k/5_types/lesson_9/lambda.k]: I doubt the first set of backticks are needed either.

Comment (laurayuwen) [inline k-distribution/tutorial/1_k/5_types/lesson_9/lambda.k]: @dwightguth @cos all backticks removed

**Comment by cos** [comment]
Jenkins, test this please

**Comment by cos** [comment]
Jenkins, test this please

**Pull request merged**